src/app/domain/iap/services/availableOffers.ts

```diff
@@ -0,0 +1,8 @@
+import { Platform } from 'react-native'
+
+const IOS_OFFERS = [
```

Comment:
Are those things really "service" level? Looks more like data/configuration

Reply:
They are not really "service" level, but they are highly related to availableOffers as their only goal is to get the list of available IAP offers on OS stores.

I'm not a big fan of over splitting files. This service is already less then 30lines so I don't feel the need scatter the info into more files. So I propose to keep it like this. Do you think this is acceptable?